src/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ if (WITH_GUI)
 endif()
 
 add_subdirectory(providers)
-if (NOT IOS)
+if (WITH_CRSSYNC)
   add_subdirectory(crssync)
```

Comment:
Can you move this inside?
We still want the command to be executed - with host tools - but the crssync lib/exe shouldn't be built.
Also the condition for initialisation should be cross compiling, not ios or android (see the link in the previous message)